### ISSUE TITLE
Multi-Select - Accessibility (screen reader, keyboard interactions etc.)

### ISSUE THREAD

**Issue by ericsoderberghp** [grommet/hpe-design-system#2661]

**Comment by bahriaditi** [comment]
@jcfilben, @amandadupell and I met with Bill Tipton today to test the MVP prototype of the multi-select component (from #2662). [Here is the recording](https://hpe-my.sharepoint.com/:f:/p/aditi_bahri/EullHMsd3HNDguF-PXkUfXcBSKbm67jQyPAUBLRz2zsEWg?e=JDjU8m) of our meeting.

**Key Takeaways:**
- **Consider adding hot keys** to help user revisit their selection: For ex: Control + home is usually a hot key to jump to the top - but right now takes user to search box, instead of top of list where selected items are listed
- **Consider adding some sort of alert** when something is selected and moves to the top - that way a screenreader user would know what happened instead of being confused about missing list items/ or that something has been selected.
- **Bookmark idea** - is there a way to bookmark where the user was previously in the list, if they jump to top of list using hot key?
- **Search label** - probably needs a better label to tell him what it is - says “edit box”
- **Search** - “No matches found” - need to communicate that somehow there were no results - was confusing for Bill to not hear anything when he typed “test”
- **Search** - Needs to know how many search results were found
- **Search** - Once user hits enter, we should move focus to the first result, instead of staying in the searchbox
- **Selection with limit:** Letting the user know that they reached their selection limit

---

**Next steps:**
As discussed with Jessica, some of these considerations will be tackled in #2662, and then we will create new tickets as needed for the more complicated ones.

**Comment by halocline** [comment]
> @jcfilben, @amandadupell and I met with Bill Tipton today to test the MVP prototype of the multi-select component (from #2662). [Here is the recording](https://hpe-my.sharepoint.com/:f:/p/aditi_bahri/EullHMsd3HNDguF-PXkUfXcBSKbm67jQyPAUBLRz2zsEWg?e=JDjU8m) of our meeting.
> 
> **Key Takeaways:**
> 
> * **Consider adding hot keys** to help user revisit their selection: For ex: Control + home is usually a hot key to jump to the top - but right now takes user to search box, instead of top of list where selected items are listed
> * **Consider adding some sort of alert** when something is selected and moves to the top - that way a screenreader user would know what happened instead of being confused about missing list items/ or that something has been selected.
> * **Bookmark idea** - is there a way to bookmark where the user was previously in the list, if they jump to top of list using hot key?
> * **Search label** - probably needs a better label to tell him what it is - says “edit box”
> * **Search** - “No matches found” - need to communicate that somehow there were no results - was confusing for Bill to not hear anything when he typed “test”
> * **Search** - Needs to know how many search results were found
> * **Search** - Once user hits enter, we should move focus to the first result, instead of staying in the searchbox
> * **Selection with limit:** Letting the user know that they reached their selection limit
> 
> **Next steps:** As discussed with Jessica, some of these considerations will be tackled in #2662, and then we will create new tickets as needed for the more complicated ones.

Awesome findings/takeaways. Nice work.
